src/components/CIPipelineN/TaskDetailComponent.tsx

```diff
@@ -190,6 +197,19 @@ export function TaskDetailComponent() {
                     />
                 </div>
 
+                {
+                    activeStageName == BuildStageVariable.PostBuild &&
+                    <div className="row-container mb-12">
+                        <div className="fw-6 fs-13 lh-32 cn-7 ">Trigger even if build fails</div>
+                        <input
+                            type="checkbox"
+                            className="cursor icon-dim-16"
+                            checked={formData[activeStageName].steps[selectedTaskIndex].triggerIfParentStageFail}
+                            onChange={(e) => handleTriggerIfParentStageFailChange(e)}
```

Comment:
we can directly write onChange={handleTriggerIfParentStageFailChange}

Reply:
done